From 34b500f59e76a21ca432786e08f62487c2e89a92 Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Fri, 28 Oct 2022 15:00:15 +0300
Subject: [PATCH 02/19] board: s32gxxxardb: Simplify setting pfe1_phy_addr

Simplify setting pfe1_phy_addr for RDB3 rev. D, E
and F.

Issue: ALB-9479

Upstream-Status: Pending 

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 board/nxp/s32-cc/s32g/s32gxxxardb.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/board/nxp/s32-cc/s32g/s32gxxxardb.c b/board/nxp/s32-cc/s32g/s32gxxxardb.c
index e5efc277b4..b983577127 100644
--- a/board/nxp/s32-cc/s32g/s32gxxxardb.c
+++ b/board/nxp/s32-cc/s32g/s32gxxxardb.c
@@ -103,8 +103,6 @@ int board_late_init(void)
 			env_set("pfe1_phy_addr", "3");
 			break;
 		case 'D':
-			env_set("pfe1_phy_addr", "8");
-			break;
 		case 'E':
 		case 'F':
 			env_set("pfe1_phy_addr", "8");
-- 
2.17.1

